blog/_posts/2021-01-15-snap-announcement.md

```diff
@@ -0,0 +1,158 @@
+
+On Ubuntu, it's as simple as:
+```
+sudo snap install root-framework && root
+```
+
+You might be able to even just search for the ROOT Framework and install it in a single click!
+
+![ROOT in the Ubuntu App Store]({{ '/assets/images/Snap_Announcement_01.png' | relative_url }})
+
+This is full-fat installation of ROOT, complete with its utilities such as hadd, PyROOT via Python 3.8 (with SciPy, NumPy, Pandas and Matplotlib), and JupyROOT.
+You'll get these bundled by default, and since the whole package is based on container technology, they don't interfere with any of your system libraries and can be easily upgraded (automatically!), removed, and mixed alongside other ROOT installations.
+
+For example, just run `root` in the terminal after installation and you'll get exactly what you expected.
+Give `root --notebook` a go and try out the JupyROOT support.
+As a special case, if you want PyROOT, you'll need to run `pyroot` rather than `python`; this ensures you get the bundled version of Python in the container rather than the host system, but from there you can import ROOT normally and run your normal scripts, `pyroot -i $(root-config --tutdir)/pyroot/fillrandom.py` away! 
+
+There's no need to mount the $HOME directory, graphical support should work by default, and a lot of optional packages are built by default.
+The goal is to provide a Docker like experience for ROOT but blur the distinction between the container and host environment, in a way that's convenient for users. 
+For example, by simply adding a shortcut to start ROOT in the start menu of most systems under the science section.
+
+
+## Tell Me More
+### Sandboxing
+Most snap packages are under a sandboxing model that might subtly interfere with a users regular workflow.
+Support for this sandboxing varies per distribution but is generally being improved overtime across the entire Linux ecosystem.
+This means that the ROOT snap can't just access a users camera or microphone for example, since this makes little sense for ROOT. However, one notable feature is that ROOT will be limited to accessing files in the users home directory (aside from over the network).
+Furthermore, it'll be prevented from accessing hidden files/folders in the top level of the home directory itself, such as `$HOME/.ssh`.
+
+To help make this work, the `$HOME` variable, and `gSystem->HomeDirectory()` will return a modified value for the users home directory, generally `/home/example/snap/root-framework/current/`. If a user wants to make use of `rootlogin.c` for the entire application, keep in mind it'll look for it in $HOME which points there instead. 
```

Comment:
I've added

"If you make use of the parallel installation ability mentioned below, this can be an advantage as each installed version of the ROOT snap will have a unique home, and may have different `rootlogin` files, history, etc."